### PR TITLE
Update template values to work with grafana 9

### PR DESCRIPTION
**Pull request by marek-jiri**
Signed-off-by: Jiri Marek <jiri.marek@dnation.cloud>

### CODE REVIEW

File: jsonnet/dashboards/k8s/memory-namespace-overview.libsonnet [hunk 1]
```diff
@@ -44,7 +44,7 @@ local graphPanel = grafana.graphPanel;
           sort={ col: 8, desc: true },
           styles=[
             { pattern: 'Time', type: 'hidden' },
-            { alias: 'Pods', pattern: 'Value #A', type: 'number' },
+            { alias: 'Pods', pattern: 'Value', type: 'number' },
```
Comment (chess-knight) [inline jsonnet/dashboards/k8s/memory-namespace-overview.libsonnet:47]: This doesn't work. there are also \#B \#C...

**Comment by chess-knight** [comment]
Because the issue with the description for this PR doesn't exist, please add some description to this PR or/and some more useful commit message.

**Review by chess-knight — COMMENTED** [review]
Because the issue with the description for this PR doesn't exist, please add some description to this PR or/and some more useful commit message.

File: jsonnet/dashboards/k8s/memory-namespace-overview.libsonnet (hunk 1 above)
Comment (matofeder) [reply jsonnet/dashboards/k8s/memory-namespace-overview.libsonnet]: Fixed.

**Review by matofeder — APPROVED** [review]
LGTM

**Pull request merged**